src/posts/2022-02-10-design-het-web.md

```diff
@@ -0,0 +1,68 @@
+---
+title: 'Design het Web'
+category: inspiration
+tags: new blogpost
+comments: true
+author: joris
+min_read: 5
+date: 2022-02-10
```

Comment:
Je kan overwegen om de datum op maandag 14 feb te zetten (ook bij permalink). Dan kunnen we ervoor zorgen dat we je blog maandag publiseren :-)